canonical/ubuntu.com
# /containers copy updates

## Done

- Applied changes as per [doc](https://docs.google.com/document/d/1OtyfNeA--EJgpgvDzqcbJ8jqa6d9H5H9xyxHNxcKXvc/edit)

## QA

- View the site locally in your web browser at: http://0.0.0.0:8001/containers 
- See that the requested changes have been made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12191
